components/TaskCard.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <div class="card">
+      <div class="cardDescription">{{ cardData.description }}</div>
+      <div style="margin-top:9px; margin-bottom:26px;" class="caption">{{ cardData.createdAt }}</div>
+      <div style="display:flex; gap:8px;">
+        <div>
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="19" viewBox="0 0 24 19" fill="none">
+            <path d="M20.285 0L9 11.567L3.714 6.556L0 10.272L9 19L24 3.715L20.285 0Z" fill="#BBBDD0"/>
+          </svg>
+        </div>
+        <div>
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+            <path d="M7.127 22.562L0 24L1.438 16.872L7.127 22.562ZM8.541 21.148L19.769 9.923L14.079 4.231L2.852 15.458L8.541 21.148ZM18.309 0L15.493 2.817L21.184 8.508L24 5.689L18.309 0V0Z" fill="#BBBDD0"/>
+          </svg>
```

Comment:
We can create separate component for the SVG icons and use them so that our markup is minimal and friendly to first look.

Reply:
Resolved